packages/kbn-i18n/README.md

```diff
@@ -46,10 +46,14 @@ themselves, and those messages will always be in English, so we don't have to ke
 defined inline.
 
 __Note:__ locale defined in `i18n.locale` and the one used for translation files should
-match exactly, e.g. `i18n.locale: zn` and `.../translations/zh_CN.json` won't match and
-default English translations will be used, but `i18n.locale: zh_CN` and`.../translations/zh_CN.json`
+match exactly, e.g. `i18n.locale: zn` and `.../translations/zh-CN.json` won't match and
+default English translations will be used, but `i18n.locale: zh-CN` and`.../translations/zh-CN.json`
 or `i18n.locale: zn` and `.../translations/zn.json` will work as expected.
 
+__Note:__ locale should look like `zh-CN` where `zh` - lowercase two-letter or three-letter ISO-639 code
+and `CN` - uppercase two-letter ISO-3166 code. (optional)
```

Comment:
__typo:__ `ISO-3166 code. (optional)` --> `ISO-3166 code (optional). ` To make it clear what is `optional`.